buku

```diff
@@ -2396,7 +2424,7 @@ class BukuDb:
     def auto_import_from_browser(self):
         """Import bookmarks from a browser default database file.
 
-        Supports Firefox and Google Chrome.
+        Supports Microsoft Edge, Firefox and Google Chrome.
```

Comment:
Please add the MS Edge cases at the end in the comments and code. Firefox and Chrome are far more popular browsers. I am not sure why we need to add Edge as the top priority.

Reply:
Sorry for the misunderstanding. My intention was to order by the alphabet. As noted above I applied the requested changes